src/hooks/useReportScrollManager/index.native.ts

```diff
@@ -1,27 +1,32 @@
 import {useContext, useCallback} from 'react';
-import {ActionListContext} from '../../pages/home/ReportScreenContext';
+import {ActionListContext, ActionListContextType} from '../../pages/home/ReportScreenContext';
 
-function useReportScrollManager() {
+type ReportScrollManagerData = {
```

Comment:
`useReportScrollManager` is a platform specific implementation, create types.ts file with a shared interface for both index.ts and index.native.ts

Reply:
Done!